packages/rs-drive-abci/src/platform_types/platform_state/v0/mod.rs

```diff
@@ -252,6 +253,31 @@ impl TryFrom<PlatformStateV0> for PlatformStateForSavingV1 {
     }
 }
 
+impl TryFrom<PlatformStateV0> for ReducedPlatformStateForSavingV0 {
+    type Error = Error;
+
+    fn try_from(value: PlatformStateV0) -> Result<Self, Self::Error> {
+        Ok(ReducedPlatformStateForSavingV0 {
+            genesis_block_info: value.genesis_block_info,
+            last_committed_block_info: value.last_committed_block_info,
```

Comment:
This gonna be for previous block. PlatformState will have it for current block. Not sure it's a good idea.

Reply:
No, regarding state sync we are fine.